client/src/router/index.js

```diff
@@ -1,6 +1,14 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
 import Home from '../views/Home.vue'
+import EditUserInfo from '../components/SettingsPage/SettingsPageTabs/EditUserInfo.vue'
+import EditPreferences from '../components/SettingsPage/SettingsPageTabs/EditPreferences.vue'
+import EditFollowers from '../components/SettingsPage/SettingsPageTabs/EditFollowers.vue'
+import EditLiked from '../components/SettingsPage/SettingsPageTabs/EditLiked.vue'
+import EditPassword from '../components/SettingsPage/SettingsPageTabs/EditPassword.vue'
+import EditPrivacy from '../components/SettingsPage/SettingsPageTabs/EditPrivacy.vue'
+import EditSaved from '../components/SettingsPage/SettingsPageTabs/EditSaved.vue'
+import DeleteAccount from '../components/SettingsPage/SettingsPageTabs/DeleteAccount.vue'
```

Comment:
For subroutes rename atleast their parent folders from SettingsPage/SettingsPageTabs to settings/tabs for simplicity (when components/routes are divided into folders - there is no need to keep their full name - these are preferred to be as concise as possible since autocomplete/highlight will show their path in either case)

Reply:
And move these to views/settings/tabs folder - since these are subroutes